src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -1207,6 +1224,7 @@ private async Task UpdatePullRequestAsync(InProgressPullRequest pr, List<UpdateA
             string targetRepository,
             string branch)
         {
+            Logger.LogInformation($"Get Required Updates from {branch} to {targetRepository}");
```

Comment:
[nit]
```suggestion
            Logger.LogInformation($"Getting Required Updates from {branch} to {targetRepository}");
```

Most other log statements are written in this tense, as they indicate that an operation is ongoing